docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+# Use postgres/example user/password credentials
+version: '3.8'
+
+services:
+  deep-lynx-postgres:
+    build:
+      context: .
+      dockerfile: ./Dockerfiles/PostgreSQL/Dockerfile
+    ports:
+      - 5432:5432
+    restart: always
+    volumes:
+      - ./Dockerfiles/PostgreSQL/deeplynx.conf:/etc/postgresql/postgresql.conf
+      - ./Dockerfiles/PostgreSQL/basedata:/private/var/lib/postgresql/data
+      - ./Dockerfiles/PostgreSQL/InitializationScripts:/docker-entrypoint-initdb.d/
+    environment:
+      - POSTGRES_PASSWORD=#{PostgresPassword}
```

Comment:
The variable replacement `#{PostgresPassword}` syntax is Octopus specific. This just needs to be replaced with whatever build and deploy syntax we eventually go with